tests/sqllogictests/suites/base/09_fuse_engine/09_0026_merge_into.test

```diff
@@ -622,6 +622,32 @@ select a, b, c from tt2 order by b;
 0 20 {'k2':'v2'}
 1 30 {'k1':'v1'}
 
+## add test: source is table
+statement ok
+drop table if exists t1;
+
+statement ok
+drop table if exists t2;
+
+statement ok
+create table t1(a int);
+
+statement ok
+create table t2(a int);
+
+statement ok
+insert into t1 values(1);
+
+statement ok
+insert into t2 values(1),(2);
+
+statement ok
+merge into t1 using t2 on t1.a = t2.a when matched then delete when not matched then insert *;
```

Comment:
Will an error be reported if the table fields of `t1` and `t2` are different? It would be good to add some tests of this type.

Reply:
ok